docs/develop.rst

```diff
@@ -1,29 +1,92 @@
 Development
 ===========
 
-SQL files
----------
+.. _devenv:
 
-The SQL files in the `sql directory <https://github.com/open-contracting/kingfisher-views/tree/master/sql>`__ are run by the :ref:`refresh-views` command.
+Development environment
+-----------------------
 
-Run specific files
-~~~~~~~~~~~~~~~~~~
+You need to set up the appropriate development environment to modify Kingfisher Views. You can install the requirements locally, or use the preconfigured Vagrant setup.
 
-If you're working on a specific file, you can run it on its own, using the ``psql`` command. For example, to run the ``004-planning.sql`` file in the ``view_data_the_name`` schema, using the :ref:`default database connection settings<database-connection-settings>`:
+Locally, you will need a Postgres installation, as well as `Kingfisher Process <https://kingfisher-process.readthedocs.io/en/latest/requirements-install.html>`__ and `Kingfisher Views <https://kingfisher-views.readthedocs.io/en/latest/get-started.html#install-kingfisher-views>`__ installed in a virtual environment.
+
+We recommend using the Vagrant setup though. `These instructions will get you started with Vagrant for Kingfisher <https://ocdskingfisher.readthedocs.io/en/latest/vagrant.html#>`__. When you are modifying SQL files or source code for Kingfisher Views, do this on your host machine, *not* in the Vagrant environment.
+
+You probably already have a preferred client for working with Postgres databases. The database name, user and password are all :code:`ocdskingfisher`.
+
+Notes on Postgres clients:
+  * To use `psql` inside the Vagrant box, solve permissions issues by running :code:`createuser -s vagrant`
+  * To use a client like pgadmin with Vagrant, connect on port 7070.
```

Comment:
I personally wouldn't put this in here - it's duplicating information that's in the KF repository/docs, and the risk is that things will change there and leave this out of date. Instead, I'd link directly to an applicable heading in the KF docs - once https://github.com/open-contracting/kingfisher/pull/345 is merged there will be one. But then I'm not sure of discussions at docs day?